docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+    ```java
+    java -jar virtual-usb-client-2.0.0.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName Motorola_Moto_Z_real --tunnel-identifier awesometunnel
+    ```
 
-```java
-adb connect localhost:7000
-```
+7. If your vUSB test session launch is successful, you'll see a success message containing:
 
-9. **Android Only**: Now, your vUSB should be fully connected. You can use Android Studio (or Google Chrome's Remote Debugging) to debug your app, execute automation based on adb, or any other tool that is adb-compliant. For example, using adb shell, you can start the camera of the connected device:
+  <Tabs
+    defaultValue="Android"
+    values={[
+      {label: 'Android', value: 'Android'},
+      {label: 'iOS', value: 'iOS'},
+    ]}>
 
-```java
-adb shell
-am start -a android.media.action.IMAGE_CAPTURE
-```
+  <TabItem value="Android">
 
-10. **Android Only**: Run your tests. Remember, you can use the `--help` flag to view a full CLI reference guide for vUSB functionality.
+  Your **session ID**, a **port number**, and a **link** that takes you to your device and allows you to watch your test running in real time.
 
-11. **Android Only**: When you're ready to close your vUSB connection/session, disconnect your device from ADB by running `adb disconnect` followed by your `<IPAddress>:<portNumber>`:
+  ```java
+  37D274BC3A65A34BB3DA4DDF7B77E341        Motorola Moto Z     ANDROID     7.0     https://app.us-west-1.saucelabs.com/live/mobile/dataCenters/US/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a
 
-```java
-adb disconnect localhost:7000
-```
+  localhost:7000  online
+  ```
+
+  </TabItem>
+  <TabItem value="iOS">
+
+  Your **session ID** and a **link** that takes you to your device and allows you to watch your test running in real time (no port number for iOS).
+
+  ```java
+  37D274BC3A65A34BB3DA4DDF7B77E341        iPhone XS             IOS         14.3    https://app.us-west-1.saucelabs.com/live/mobile/dataCenters/US/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a
+
+  localhost:-1  online
+  ```
+
+  </TabItem>
+  </Tabs>
+
+8. Open your test:
+
+ <Tabs
+   defaultValue="Android"
+   values={[
+     {label: 'Android', value: 'Android'},
+    {label: 'iOS', value: 'iOS'},
+   ]}>
+
+ <TabItem value="Android">
+
+ Link ADB to your test session device by running `adb connect`, followed by the port number:
+  ```java
+  adb connect localhost:7000
+  ```
+  <img src={useBaseUrl('img/virtual-usb/vusb-adb.jpg')} alt="Virtual USB ADB" width="400" />
+
+ Open up your device by running the URL from your success message in a browser.
 
-12. Close the session through the command line.
+ </TabItem>
+ <TabItem value="iOS">
 
+  Open up your device by running the URL from your success message in a browser. This will prompt an authentication pop-up window, where you'll need to provide Touch ID or password authentication.
```

Comment:
This is not correct, this happens already in step 7 for iOS. there are no additional steps for iOS other then opening the url in a browser

Reply:
all set -- 


7. If your vUSB test session launch is successful, you'll see a success message containing:

  <Tabs
    defaultValue="Android"
    values={[
      {label: 'Android', value: 'Android'},
      {label: 'iOS', value: 'iOS'},
    ]}>

  <TabItem value="Android">

  Your **session ID**, a **port number**, and a **link**. Make sure you're logged into your Sauce Labs account, then click the link to see your device in action, running your tests in real time.

  ```java
  37D274BC3A65A34BB3DA4DDF7B77E341        Motorola Moto Z     ANDROID     7.0     https://app.us-west-1.saucelabs.com/live/mobile/dataCenters/US/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a

  localhost:7000  online
  ```

  </TabItem>
  <TabItem value="iOS">

  Your **session ID** and a **link** (no port number for iOS). Make sure you're logged into your Sauce Labs account, then click the link. This will prompt an authentication pop-up window, where you'll need to provide Touch ID or password authentication. Then, you can see your device in action, running your tests in real time.

  ```java
  37D274BC3A65A34BB3DA4DDF7B77E341        iPhone XS             IOS         14.3    https://app.us-west-1.saucelabs.com/live/mobile/dataCenters/US/devices/shared/9299h0c88a7-e2b6-41bc-9509-5-8a5d765490371e2c9a

  localhost:-1  online
  ```

  </TabItem>
  </Tabs>


8. **Android only**: Link ADB to your test session device by running `adb connect`, followed by the port number:

  ```java
  adb connect localhost:7000
  ```
  <img src={useBaseUrl('img/virtual-usb/vusb-adb.jpg')} alt="Virtual USB ADB" width="400" />

Reply:
@kimsauce 

Yes, this is better, thanks